openshift/templates/visual-verifier/visual-verifier-deploy.safe-entry.param

```diff
@@ -5,15 +5,18 @@
 #=========================================================
 NAME=visual-verifier
 APP_NAME=visual-verifier
-APP_GROUP=safe-entry
-SUFFIX=-safe-entry
+APP_GROUP=visual-verifier
+SUFFIX=
 ROLE=visual-verifier
-APPLICATION_DOMAIN=safe-entry-verifier-dev.pathfinder.gov.bc.ca
+APPLICATION_DOMAIN=visual-verifier-bztwou-dev.pathfinder.gov.bc.ca
```

Comment:
These need to be reverted to their previous values, otherwise it will cause issues when re-deploying the verifier (suffix and app group will not match, and the templates will result in a "different app") being processed